docs/install/aws-install.md

```diff
@@ -12,7 +12,8 @@ AWS account with Permission to run a CloudFormation stack, create an EKS cluster
 - You will need to have a route53 domain configured. External domain registration is currently not supported. For more information see [What is Amazon Route 53](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/Welcome.html).
```

Comment:
- You need to have a route53 domain configured. External domain registration is currently not supported. For more information see [What is Amazon Route 53](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/Welcome.html).